### PR TITLE
welcome & servers update

### PR DESCRIPTION
**Pull request by hepcat72**

### CODE REVIEW

File: roles/galaxy/templates/config/galaxy.yml.j2 [hunk 1]
```diff
@@ -69,3 +68,4 @@ galaxy:
   enable_quotas: true
   tool_search_limit: 40
   py-call-osafterfork: true
+  conda_ensure_channels: iuc,conda-forge,bioconda,defaults
```
Comment (lparsons) [inline roles/galaxy/templates/config/galaxy.yml.j2:71]: Q: Why do we need to set the conda channels now?

File: development/group_vars/galaxy_servers [hunk 2]
```diff
@@ -9,7 +9,7 @@ pip_virtualenv_command: /usr/bin/python3 -m venv
 
 galaxy_changeset_id: release_21.05
 config_directory: /galaxy/galaxy-app/config
-crontab_mailto: lparsons@princeton.edu
+crontab_mailto: lparsons@princeton.edu,rleach@princeton.edu
```
Comment (lparsons) [inline development/group_vars/galaxy_servers:12]: Q: Were you able to confirm that this worked. In other words, do we both get crontab emails?

**Review by lparsons — COMMENTED** [review]
Looks good, lots of updates! A had a couple of questions inline. Also, I'd like to just do a quick preview of the welcome page updates on galaxy-dev, but this doesn't seem to be pushed out there yet. Could you push the welcome page to dev just we can see it rendered?

**Comment by hepcat72** [comment]
Having trouble booting up my office computer this morning. On my phone now. I'll follow up once I'm on.

File: roles/galaxy/templates/config/galaxy.yml.j2 (hunk 1 above)
Comment (lparsons) [reply roles/galaxy/templates/config/galaxy.yml.j2:71]: Is it not sufficient to use the default setting?

Comment (hepcat72) [reply roles/galaxy/templates/config/galaxy.yml.j2:71]: I was being safe.  The new default had removed one. `iuc`, if I recall correctly.

Comment (lparsons) [reply roles/galaxy/templates/config/galaxy.yml.j2:71]: I believe we should remove this line and continue to use the defaults (which are now changed). From the [release notes](https://docs.galaxyproject.org/en/latest/releases/21.01_announce.html#deprecation-notices):
> The IUC conda channel has been removed from the default configuration. This channel was used for installing dependencies that were old and not available in bioconda or conda-forge. These dependencies are now available in bioconda. By removing this channel Conda can work more efficiently and ignore outdated dependencies. If you have specified alternative channels in your Galaxy configuration we recommend removing the iuc channel from the conda_ensure_channels variable.

Comment (hepcat72) [reply roles/galaxy/templates/config/galaxy.yml.j2:71]: Sure.  I created issue #6

**Pull request merged**